cpp/lib/src/ZoteroHarvesterConfig.cc

```diff
@@ -265,15 +265,44 @@ const std::map<JournalParams::IniKey, std::string> JournalParams::KEY_TO_STRING_
     { CRAWL_URL_REGEX,         "zotero_crawl_url_regex" },
 };
 
+const std::map<std::string, JournalParams::IniKey> JournalParams::STRING_TO_KEY_MAP {
+    { "zeder_id",                   ZEDER_ID },
+    { "zeder_modified_time",        ZEDER_MODIFIED_TIME },
+    { "zotero_group",               GROUP },
+    { "zotero_url",                 ENTRY_POINT_URL },
+    { "zotero_type",                HARVESTER_OPERATION },
+    { "zotero_delivery_mode",       UPLOAD_OPERATION },
+    { "online_ppn",                 ONLINE_PPN },
+    { "print_ppn",                  PRINT_PPN },
+    { "online_issn",                ONLINE_ISSN },
+    { "print_issn",                 PRINT_ISSN },
+    { "zotero_strptime_format",     STRPTIME_FORMAT_STRING },
+    { "zotero_update_window",       UPDATE_WINDOW },
+    { "zotero_review_regex",        REVIEW_REGEX },
+    { "zotero_expected_languages",  EXPECTED_LANGUAGES },
```

Comment:
I apologize for nitpicking.  The minimal number of spaces in such a tabular arrangement should be 1.  Also we usually like up the closing braces on each line, too.  See also the previous map initialization above.